technology-overview.md

```diff
@@ -0,0 +1,81 @@
+# Technology overview
+
+[Agoric](https://agoric.com/about/) aims at enabling safe cooperation among otherwise mutually suspicious parties
+
+The way it does it is by making it easy to write secure smart contracts and making it possible to run these contracts on one or several blockchains. Agoric strategy to writing smart contracts is to write them in JavaScript using a framework called ERTP (Electronic Rights Transfer Protocol)
+
+
+## High-level view
+
+### JavaScript
+
+Agoric works to make [ECMAScript](https://www.ecma-international.org/publications/standards/Ecma-262.htm) (standard defining JavaScript) a suitable language for easily expressing secure smart contracts. Agoric works with ECMA TC39 (which includes browser makers and Node.js representatives) in that direction
+
+Agoric also works with [ECMA TC53](https://www.ecma-international.org/memento/tc53.htm) to make [SES](https://github.com/Agoric/SES) the JavaScript for embedded devices
+
+
+#### Long term
+
+- [Realms](https://github.com/tc39/proposal-realms)
+- [Eventual-send operator](https://github.com/Agoric/proposal-infix-bang)
+- [WeakRefs](https://github.com/tc39/proposal-weakrefs/)
+
+
+#### Short term
```

Comment:
The "Long term" vs "Short term" dichotomy followed by a shim as the first short term item leads to the impression that these are only short term, to be replaced eventually by long term items when they are ready. But Jessie and SES are also long term, which may be missed.

Reply:
I clarified this in the next commit
Is it ok now?

Reply:
Hi @DavidBruant , yes, that fixes that problem. Thanks.